server/index.js

```diff
@@ -73,6 +73,66 @@ app.get('/api/random', (req, res) => {
     });
 });
 
+app.get('/api/random/selected', (req, res) => {
+  res.status(201).json(selectedRestaurant);
+});
+
+app.post('/api/random/favorite', (req, res) => {
+  // eslint-disable-next-line camelcase
+  const { id, name, image_url, rating, review_count } = selectedRestaurant;
+  const categories = selectedRestaurant.categories.map(category => category.title).join(' ');
+  const userId = 1;
+
+  const address = `${selectedRestaurant.location.address1}, ${selectedRestaurant.location.city} ${selectedRestaurant.location.state} ${selectedRestaurant.location.zip_code}`;
+
+  const sql = `
+            insert into "favorites" ("businessId", "restaurantName", "imageUrl", "rating", "totalReviews", "address", "categories", "userId")
+            values ($1, $2, $3, $4, $5, $6, $7, $8)
+            returning *
+          `;
+  // eslint-disable-next-line camelcase
```

Comment:
You can turn this rule off in your `.eslintrc` so you don't have to keep adding this comment.